### PR TITLE
Init react and angular frontend

### PR DESCRIPTION
**Pull request by nateng98**
skeletons for both frontend (react & angular) created, ready for to be merged into dev

### CODE REVIEW

**Review by Mondane-Monkeys — APPROVED** [review]
Looks really solid. Ready to merge.

Some feedback:
- Readme should contain build and run instructions 
- Dockerization would be good
- Setting up a deployment solution for guest would be good.

In an ideal world, where we have infinite time, skeletons should be in a "ready to run, just needs content" state. In this case you definitely made the right call, getting it committed quickly is more important than getting everything resolved.

TL;DR, great work! Approved from me 😁
